pandas/core/internals.py

```diff
@@ -6,6 +6,8 @@
 from collections import defaultdict
 
 import numpy as np
+from numpy import percentile as _quantile
```

Comment:
I think maybe move this to ``compat`` as in numpy > 1.9 we can import ``nanpercentile`` (not sure we have enough test coverage, maybe make another issue about this and can fix later)